### ISSUE TITLE
Luarocks installation not working

### ISSUE THREAD

**Issue by ErickProgramer** [teal-language/teal-language-server#37]
Hi guys, I wanna know how to install this language server in neovim, I used luarocks to install and with the Mason as well and also I made: `require("lspconfig").teal_ls.setup{}`, but this server does NOT work at all. I give up. I'd appreciate if somebody could help me.

**Comment by ErickProgramer** [comment]
After many time I did it. (pls improve the documentation because it's painful)

**Comment by svermeulen** [comment]
What was the actual problem and fix?

**Comment by chtk** [comment]
I'd like to know this as well.

Like you, @ErickProgramer, I've been trying to get teal-language-server installed though Mason. It's be great if you could describe here what you did at least. Or maybe even push a PR for the docs.

**Comment by ErickProgramer** [comment]
> What was the actual problem and fix?

Basically, I tried to install it with Mason, and I had previously executed luarocks install teal-language-server, but it was not working at all. Mason was displaying an error saying it was trying to use the --dev flag in luarocks to install the server. The final problem (I think) was that teal_ls.lua was not in Neovim (I don’t know why), so I manually added it.
